plugins/modules/source_control/gitlab/gitlab_service.py

```diff
@@ -0,0 +1,468 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Raphaël Droz <raphael@droz.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: gitlab_service
+short_description: Setup or delete GitLab integration services
+version_added: '2.4.0'
+description:
+  - Creates, updates or deletes GitLab services (aka "integrations").
+author:
+  - Raphaël Droz (@drzraf)
+requirements:
+  - python >= 2.6
```

Comment:
```suggestion
```

Reply:
I've waiting for years for https://github.com/ansible/ansible/projects/32 to be closed. But if you suggest this you'd have to tell me twice.

Reply:
I don't understand how that project is related to this change. Python 2.6 is still supported, and Python 2.5 has not been supported for some time, so stating that this (or any) module requires Python 2.6 or newer does not make sense (and thus can be removed).

Reply:
Removed. (but a lot of modules contains this)